.github/actions/setup-project/action.yml

```diff
@@ -47,11 +42,6 @@ runs:
       shell: bash
       run: pnpm install --frozen-lockfile
 
-    - name: Link submodules
```

Comment:
why this part not needed anymore?

Reply:
I am not sure why it was needed in the first place. Let's see if the actions run.

Reply:
i am a bit confused, this is needed to create a symlink from the `enterprice/packages/*/src` folder to the `/.source/*/src` folder.

Reply:
The link is committed in the repo via the `git submodule` configuration.